cmd/notation/blob/policy/import.go

```diff
@@ -34,11 +34,14 @@ func importCmd() *cobra.Command {
 	var opts importOpts
 	command := &cobra.Command{
 		Use:   "import [flags] <file_path>",
-		Short: "Import trust policy configuration from a JSON file",
+		Short: "Import blob trust policy configuration from a JSON file",
 		Long: `Import blob trust policy configuration from a JSON file.
 
-Example - Import trust policy configuration from a file:
+Example - Import blob trust policy configuration from a file:
   notation blob policy import my_policy.json
+
+Example - Import blob trust policy and override existing configuration without prompt:
+  notation blob policy import my_policy.json --force
```

Comment:
nit: put flags before the file path since it is example
```suggestion
  notation blob policy import --force my_policy.json
```

Reply:
Updated.